client/src/components/campaign/round/round.component.js

```diff
@@ -0,0 +1,111 @@
+import template from './round.html';
+
+const Component = {
+  bindings: {
+    campaign: '=',
+    last: '=',
+    round: '=',
+  },
+  controller,
+  template,
+};
+
+function controller(
+  adminService,
+  alertService) {
+  const vm = this;
+
+  vm.activateRound = activateRound;
+  vm.finalizeRound = finalizeRound;
+  vm.pauseRound = pauseRound;
+  vm.populateRound = populateRound;
+  vm.editRound = () => { vm.round.edit = true; };
+
+  // functions
+
+  vm.$onInit = () => {
+    getRoundDetails(vm.round);
+    getRoundResults(vm.round);
+  };
+
+  /**
+   * 
+   */
+  function activateRound() {
+    adminService
+      .activateRound(vm.round.id)
+      .then(() => {
```

Comment:
This probably needs a loading spinner, since it can take a while for bigger rounds.